lang/src/lib.rs

```diff
@@ -82,6 +82,7 @@ pub trait Accounts<'info>: ToAccountMetas + ToAccountInfos<'info> + Sized {
         accounts: &mut &[AccountInfo<'info>],
         ix_data: &[u8],
         bumps: &mut BTreeMap<String, u8>,
+        reallocs: &mut BTreeSet<String>,
```

Comment:
Why is this a set of `String` instead of `Pubkey`?

Reply:
fixed for next commit.